docroot/modules/custom/uiowa_aggregator/uiowa_aggregator.module

```diff
@@ -174,8 +174,14 @@ function _uiowa_aggregator_get_feeds() {
   $feeds = \Drupal::entityTypeManager()->getStorage('aggregator_feed')->loadMultiple();
   $options = [];
 
-  foreach ($feeds as $feed) {
-    $options[$feed->id()] = $feed->label();
+  if ($feeds) {
+    foreach ($feeds as $feed) {
+      $options[$feed->id()] = $feed->label();
+    }
+  }
+  else {
+    $options = ['no-results' => 'No feeds available'];
```

Comment:
This would allow people to actually select `No feeds available` from the list and not render anything. I think that'd be odd and IMO the message below would be enough.

Reply:
Noticed that ☹️ COOL about the error message though!